src/processes/filters.rs

```diff
@@ -43,14 +43,14 @@ impl QueryFilter {
 
     pub(super) fn accept(&self, prc: &impl ProcessInfo, ports: Option<&str>) -> bool {
         match self.search_by {
-            SearchBy::Cmd => self.query_matches_str(prc.cmd()),
-            SearchBy::Path => self.query_matches_fuzzy_opt(prc.cmd_path()),
+            SearchBy::Cmd => self.query_match_str(prc.cmd()),
+            SearchBy::Path => self.query_matches_opt(prc.cmd_path()),
             SearchBy::Args => self.query_matches_vec(get_process_args(prc)),
             SearchBy::Port => self.query_matches_opt(ports),
```

Comment:
I'm wondering why the test for ports doesn't fail on Mac when the rest of them do.

Reply:
Whats wrong with this OS for real, how adding fuzzy search can change it's behavior.
I must take a look at it later.

Reply:
It is crazy! :) Can fuzzy search work differently on Mac for some reason? :) Sounds weird, though.